src/app/components/charts/Tooltip.tsx

```diff
@@ -28,7 +28,7 @@ export const TooltipContent = ({ active, payload, formatters }: TooltipContentPr
   if (!active || !payload || !payload.length) {
     return null
   }
-  const { [payload[0].dataKey!]: value, ...rest } = payload[0]?.payload
+  const { [payload[0].dataKey!]: value, ...rest } = payload[0] ? payload[0].payload : undefined
```

Comment:
I don't understand eslint's complaint here, but `= payload[0].payload` should directly work

or, if there wasn't an un-readable `!payload.length` above, then `payload[0] ? payload[0].payload : {}` should be more accurate

Reply:
> I don't understand eslint's complaint here, but `= payload[0].payload` should directly work

Indeed, it does. Simplified.